src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
```

Comment:
the package name in java should be like "core.base_syntax"
https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html